File: shared/domain/models/visit_edit.go
package models

import "time"

// VisitRecordEdit は訪問記録の編集履歴。
type VisitRecordEdit struct {
	ID            string    `json:"id"`
	VisitRecordID string    `json:"visitRecordId"`
	EditorID      string    `json:"editorId"` // 編集者のDID
	OldBody       string    `json:"oldBody"`  // 変更前のJSON snapshot
	NewBody       string    `json:"newBody"`  // 変更後のJSON snapshot
	EditedAt      time.Time `json:"editedAt"`
}

File: pwa/src/domain/models/visit-edit.ts
// 訪問記録の編集履歴。
// 参照実装: shared/domain/models/visit_edit.go

export interface VisitRecordEdit {
  id: string;
  visitRecordId: string;
  /** 編集者のDID */
  editorId: string;
  /** 変更前のJSON snapshot */
  oldBody: string;
  /** 変更後のJSON snapshot */
  newBody: string;
  /** ISO 8601 */
  editedAt: string;
}
